ReactiveTask/Task.swift

```diff
@@ -48,6 +48,13 @@ public struct Task {
 
 extension Task: CustomStringConvertible {
 	public var description: String {
+		let arguments = self.arguments.map { argument -> String in
+			if argument.containsString(" ") {
```

Comment:
It would be better to use `NSCharacterSet` to test for whitespace—that will catch other whitespace characters like `\t`.